lesson29/src/js/resetpassworddone.js

```diff
@@ -0,0 +1,9 @@
+const checkUrlParams = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const token = localStorage.getItem("token");
+  if (urlParams.get("token") === token) {
+    window.location.href = "./resetingpassword.html";
+  }
+}
+window.location.search && checkUrlParams();
```

Comment:
could you move token check function to `utils/tokencheck.js` because of usefull everywhere.
There is written some place as same code.

Reply:
Thank you.
Moved token check function to `utils/tokencheck.js` when email/password change is completed.

[842271a](https://github.com/sae-github/JS_lesson/pull/34/commits/842271ac9a1c1ba6204b37a8f8a4d0dee1ad41af)